client/src/crypto/native.rs

```diff
@@ -1,6 +1,6 @@
 use std::{
     ops::{Mul, Rem, Shr},
-    sync::LazyLock,
+    sync::{LazyLock, OnceLock},
```

Comment:
This lacks an openssl implementation as well as that is what most distros will likely use the daemon with

Reply:
Will do, thanks!